sdks/python/container/run_validatescontainer_arm.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
is there any opportunity for code reuse, perhaps by parameterizing existing validatescontainer script, or moving some parts of code to gh actions?

Reply:
Tried to add some if else statement to the validatescontainer script so that both validatesContainerARM and validatesContainer use the same script. If those changes work, will remove this new script.

Reply:
As precommit tests pass, should I just change run_validatescontainer.sh and remove this new script?